external-import/mandiant/src/mandiant/__init__.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+import importlib
+import sys
+import time
+import traceback
+
+from dateutil.parser import parse
+from mandiant.api import MandiantAPI
+from pycti import OpenCTIConnectorHelper, get_config_variable
+
+MANDIANT_API_URL = "https://api.intelligence.mandiant.com"
+INITIAL_STATE = {
+    "actor": 0,
+    "malware": 0,
+    "vulnerability": 0,
+    "indicator": 0,
+    "report": 0,
+}
+
+
+class Mandiant:
```

Comment:
IMO, this class should be in its own file. The `__init__.py` file should only contain code needed for the initialization of the package and define what will be public.

Reply:
Good point. Done. Thank you for have the time to reviewing it.